modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{product-title} manages image caching with a local registry. A Single Node OpenShift (SNO) Distributed Unit (DU) is subject to bandwidth restrictions when communicating with the centralized registry. This can lead to long image download times.
+
+This is unavoidable for the initial deployment. Over time, there is a risk that `crio` will wipe `/var/lib/containers/storage` in the case of a disorderly shutdown.
+
+To address this, you need a local image registry on the SNO. This is useful in Edge scenarios where an SNO is deployed on the Far Edge. The SNO cannot access the internet, just its own resources and eventually the internal network.
```

Comment:
To address what? What is useful in Edge scenarios? Define what "this" is, because now I'm not sure if we're still talking about long download times or the risk or storage erasure.

"The SNO can only access its own resources and internal network."

Reply:
We are referring to long image download times here.